### ISSUE TITLE
Error in log

### ISSUE THREAD

**Issue by kirush0280** [JimStar/reolink_cctv#41]
**Debug log would be very helpful**
2022-10-30 08:05:10.055 WARNING (MainThread) [reolink_ip.api] Host 192.168.100.243:80: subscription request got a response with wrong HTTP status 400: Bad Request
2022-10-30 08:05:10.058 WARNING (MainThread) [reolink_ip.api] Host 192.168.100.241:80: subscription request got a response with wrong HTTP status 400: Bad Request
2022-10-30 08:05:10.370 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 151, in _handle_refresh_interval
    await self._async_refresh(log_failures=True, scheduled=True)
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 283, in _async_refresh
    self.async_update_listeners()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 110, in async_update_listeners
    update_callback()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 348, in _handle_coordinator_update
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 533, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 570, in _async_write_ha_state
    available = self.available  # only call self.available once per update cycle
  File "/config/custom_components/reolink_cctv/binary_sensor.py", line 150, in available
    return self._host.api.session_active and (self._host.api.subscribed or self.is_on())
TypeError: 'bool' object is not callable
2022-10-30 08:05:10.410 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 151, in _handle_refresh_interval
    await self._async_refresh(log_failures=True, scheduled=True)
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 283, in _async_refresh
    self.async_update_listeners()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 110, in async_update_listeners
    update_callback()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 348, in _handle_coordinator_update
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 533, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 570, in _async_write_ha_state
    available = self.available  # only call self.available once per update cycle
  File "/config/custom_components/reolink_cctv/binary_sensor.py", line 150, in available
    return self._host.api.session_active and (self._host.api.subscribed or self.is_on())
TypeError: 'bool' object is not callable
```

**Screenshots**
![image](https://user-images.githubusercontent.com/13412942/198863502-19dc86f0-cf47-4e22-9740-75eb0c2eaa89.png)

**Environment:**
Home Assistant 2022.10.5
Supervisor 2022.10.0
Operating System 9.3
Frontend 20221010.0 - latest
reolink_cctv - latest

**Comment by JimStar** [comment]
Fixed in [0.0.22](https://github.com/JimStar/reolink_cctv/releases/tag/0.0.22).

**Comment by kirush0280** [comment]
Thanks a lot
